src/packer.pkr.hcl

```diff
@@ -87,9 +93,10 @@ source "amazon-ebs" "windows" {
   kms_key_id                  = var.build_region_kms
   launch_block_device_mappings {
     delete_on_termination = true
-    device_name           = "/dev/xvda"
+    device_name           = "/dev/sda1"
     encrypted             = true
-    volume_size           = 8
+    no_device             = false
+    volume_size           = 500
```

Comment:
I just want to confirm that since the rework this is still an appropriate volume size (not too large).

Reply:
@mcdonnnj With the latest instance I generated took up about 70 GB, I decided to settle on a `volume_size` of 100. Resolved in [4f29107](https://github.com/cisagov/windows-commando-vm-packer/pull/21/commits/4f2910717b963e12e40a25a4da1c867ad4eee3be)